yast2_ntp-client_support_server-bsc1074726-20180104.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": [],
+  "area": [
+    {
+      "xpos": 16,
+      "ypos": 350,
+      "width": 56,
+      "height": 38,
+      "type": "match"
+    },
+    {
+      "xpos": 303,
+      "ypos": 366,
+      "width": 410,
+      "height": 19,
+      "type": "match"
+    }
+  ],
+  "tags": [
+    "bsc#1074726"

```

Comment:
If this is a workaround needle for a bug then it should have the according property

Reply:
What property? I don't understand :)
I use this tag in the `yast2_ntpclient` code. Or do I need an boo# number nstead of a bsc#?

Reply:
see https://progress.opensuse.org/projects/openqatests/wiki/#applying-quotworkaroundquot-needles . In the needle editor you can click the "workaround" checkbox which will fill an entry to "properties". Your needle name is fine. bsc# is fine. It most likely will not be visible to non-SUSE users anyway.

Reply:
OK, I will try to figure out how to do this :)